ERCS/erc-9999.md

```diff
@@ -0,0 +1,39 @@
+---
+eip: xxxx
```

Comment:
An ERC requires a minimum of a specification to be issued a number.

I suggest closing this PR and open a new one when you have a draft ready, otherwise ping when you have a specification for a number to be issued.